### PR TITLE
Propose SafetyNet as an extension

### PR DESCRIPTION
**Pull request by gmandyam**
***
<a href="https://pr-preview.s3.amazonaws.com/gmandyam/webauthn/pull/1010.html" title="Last updated on Jul 25, 2018, 4:56 PM GMT (f1ffeee)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/webauthn/1010/8b0eb71...gmandyam:f1ffeee.html" title="Last updated on Jul 25, 2018, 4:56 PM GMT (f1ffeee)">Diff</a>

### CODE REVIEW

**Comment by gmandyam** [comment]
@selfissued 

As per the attestation definition in the terminology section - "In the WebAuthn context, attestation is employed to attest to the provenance of an authenticator and the data it emits".  

SafetyNet does not attest the provenance of an authenticator or its data, therefore in my opinion it is not really a Webauthn attestation.

It does provide information on platform health, and therefore could be used to enhance other types of attestation.

**Comment by christiaanbrand** [comment]
I've left my comments here in the original issue: https://github.com/w3c/webauthn/issues/1011

TL;DR: I vote for this staying as a form of attestation.

**Comment by akshayku** [comment]
I vote to keep it as it is.

**Comment by akshayku** [comment]
Decided on 16th Jan 2019 to keep it as it is.
